server/utils/helpers.js

```diff
@@ -15,6 +20,24 @@ const { SECRET } = env;
 export const generateToken = (payload, expiresIn = '7d') =>
   jwt.sign(payload, SECRET, { expiresIn });
 
+/**
+ * Generates account verification and auth tokens
+ *
+ * @param {object} options - token generation options.
+ * @param {string} options.payload - payload to sign
+ * @param {string} options.tokenExpiresIn - auth token expiry
+ * @param {string} options.verificationTokenExpiresIn - verification token expiry
+ * @returns {Promise.<[string,string]>} [authToken, verificationToken]
+ */
+export const authTokens = async ({
```

Comment:
Why a single function that returns both?

Reply:
Convenience,  seeing as both are needed in the same controller.

Reply:
I'm not really sure how it is a convenience to generate _both_ with a single function call that requires arguments to be passed for both even though they're not both being used because auth and verification tokens are used in the same file, actually.

Reply:
The arguments passed for both are completely optional.  they(arguments) only exist to allow for dynamic configuration of token lifespan without app redeployment.

Reply:
So if someone wants to create a verification token with a custom lifespan, they need to call e.g `authTokens(payload, undefined, '1h')`, which returns an array which they then destructure to take the second element as the token they want and discard the first element as they didn't actually want it in the first place but it got generated anyway.

I'm still not really sure how this is for convenience. What exactly is the use case for generating an authentication token and a verification token at the same time?

Reply:
This function has been dropped and the generation of the tokens has been decoupled in the latest commit